src/functions/supsub.js

```diff
@@ -175,7 +175,7 @@ defineFunctionBuilders({
         }
 
         // Wrap the supsub vlist in a span.msupsub to reset text-align.
-        const mclass = html.getTypeOfDomTree(base) || "mord";
+        const mclass = html.getTypeOfDomTree(base, "right") || "mord";
```

Comment:
This change isn't necessary after re-enabling the param default.

Reply:
I added it because it wasn't obvious to me when it's omitted that the default behavior is "right". Is there any *obvious* reason to make "right" the default over, say, "left"? If not, I think having it explicitly would aid readability.